core/iwasm/aot/aot_loader.c

```diff
@@ -2262,9 +2277,14 @@ load_import_funcs(const uint8 **p_buf, const uint8 *buf_end, AOTModule *module,
                     return false;
```

Comment:
better output something to indicate the error.

Reply:
added `LOG_ERROR` here.